src/ansys/fluent/core/services/health_check.py

```diff
@@ -81,6 +84,10 @@ def status(self) -> str:
             try:
                 return self.check_health()
             except Exception:
+                ex_type, ex_value, _ = sys.exc_info()
+                logger.info(
+                    f"HealthCheckService.status() caught {ex_type.__name__}: {ex_value}"
+                )
```

Comment:
Added a logging report whenever a general exception is caught here, as this was hiding relevant exceptions (see https://github.com/ansys/pyfluent/issues/1747)